### PR TITLE
[script.video.randomtv] 1.3.0

### PR DESCRIPTION
**Pull request by gmccauley**
### Description
v1.3.0 (23 February 2021)
  - Added support for Matrix / python 3

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.video.randomtv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- Update Play Count
+- Repeat Playlist
+- Shuffle On Repeat
+- Show Notifications
+- Enable Auto Stop
+- Include All TV Shows or Select TV Shows
+It will also add a context menu item so that you can play a specific show or season with RandomTV
+	</description>
+    <disclaimer lang="en_GB">My First Add-On.  Please by gentle</disclaimer>
+    <language></language>
```
Comment (enen92) [inline script.video.randomtv/addon.xml:32]: Please drop

File: script.video.randomtv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <platform>all</platform>
+    <license>GNU General Public License, v2</license>
```
Comment (enen92) [inline script.video.randomtv/addon.xml:34]: GPL-2.0-only or GPL-2.0-or-later (spdx identifiers)

File: script.video.randomtv/addon.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    <forum>https://forum.kodi.tv/showthread.php?tid=310494</forum>
+    <source>https://github.com/gmccauley/script.video.randomtv</source>
+    <news>
+v1.3.0 (23 February 2021)
```
Comment (enen92) [inline script.video.randomtv/addon.xml:38]: Please use news for the most recent changes. You can ship a changelog.txt with the whole changelog
T

**Review by enen92 — CHANGES_REQUESTED** [review]
Minors. Thanks for the update.

**Comment by gmccauley** [comment]
> Minors. Thanks for the update.

Thanks.  I've updated, but I don't know git well enough to squash them into this commit / pull request.  So I'll close this one and open another.

**Pull request closed without merging**